src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -662,6 +662,36 @@ internal static void SetCurrentThreadUiCulture(CultureInfo uiCultureInfo)
         #region Misc
 
         /// <summary>
+        /// Facade for Convert.ToBase64String(bytes, Base64FormattingOptions.InsertLineBreaks)
+        /// Inserts line breaks after every 76 characters in the string representation.
+        /// </summary>
+        internal static string ToBase64StringWithLineBreaks(byte[] bytes)
```

Comment:
So they expose the 'Base64FormattingOptions' overload in runtime assembly but not in contract assembly, interesting. Since this overload is not in contract, we cannot use it in C# code.